### PR TITLE
Port remaining tasks to Kotlin

### PR DESCRIPTION
**Pull request by benjamin-bader**

### CODE REVIEW

File: src/main/kotlin/com/getkeepsafe/dexcount/Tasks.kt [hunk 1]
```diff
@@ -29,6 +32,36 @@ import java.io.File
...
+@ParallelizableTask
+open class LegacyMethodCountTask: DexMethodCountTaskBase() {
+
+    lateinit var variantOutput: BaseVariantOutput
+
+    override val fileToCount: File?
+        get() = variantOutput.outputFile
+
+    override val rawInputRepresentation: String
+        get() = "variantOutput={name=${variantOutput.name} outputFile=${variantOutput.outputFile}}"
```
Comment (jaredsburrows) [inline src/main/kotlin/com/getkeepsafe/dexcount/Tasks.kt]: Add space here: `le = ${`

Comment (benjamin-bader) [inline src/main/kotlin/com/getkeepsafe/dexcount/Tasks.kt]: Not sure what you mean - are you suggesting changing `outputFile=${variantOutput.outputFile}` to `outputFile = ${variantOutput.outputFile}`?

Comment (jaredsburrows) [inline src/main/kotlin/com/getkeepsafe/dexcount/Tasks.kt]: Yeah, its add in the missing spaces! 👍

Comment (benjamin-bader) [inline src/main/kotlin/com/getkeepsafe/dexcount/Tasks.kt]: ok - no strong feelings here, that's just how it was before.  will do.

**Comment by jaredsburrows** [comment]
LGTM. Thanks.

**Pull request merged**